extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/AdditionalHttpServerMetricsTagsProvider.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.micrometer.runtime;
+
+import io.micrometer.core.instrument.Tags;
+import io.vertx.core.http.HttpServerRequest;
+
+/**
+ * Allows code to add additional Micrometer {@link Tags} to the metrics collected for completed HTTP server requests.
+ * <p>
+ * The implementations of this interface are meant to be registered via CDI beans.
+ */
+public interface AdditionalHttpServerMetricsTagsProvider {
```

Comment:
I very much dislike this name, so if you have ideas, I'm all ears

Reply:
it works for me

Reply:
I've updated it after talking to @ebullient